[2018-06-19 11:44:35] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /" (from "http://localhost/symfony_test/")" at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/EventListener/RouterListener.php line 139 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" (from \"http://localhost/symfony_test/\") at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/EventListener/RouterListener.php:139, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /opt/lampp/htdocs/symfony_test/var/cache/dev/srcDevDebugProjectContainerUrlMatcher.php:160)"} []
[2018-06-19 11:59:02] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /" (from "http://localhost/symfony_test/")" at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/EventListener/RouterListener.php line 139 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" (from \"http://localhost/symfony_test/\") at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/EventListener/RouterListener.php:139, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /opt/lampp/htdocs/symfony_test/var/cache/dev/srcDevDebugProjectContainerUrlMatcher.php:161)"} []
[2018-06-19 12:05:47] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /" (from "http://localhost/symfony_test/")" at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/EventListener/RouterListener.php line 139 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" (from \"http://localhost/symfony_test/\") at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/EventListener/RouterListener.php:139, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /opt/lampp/htdocs/symfony_test/var/cache/dev/srcDevDebugProjectContainerUrlMatcher.php:161)"} []
[2018-06-19 12:06:45] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://localhost/symfony_test/public/","method":"GET"} []
[2018-06-19 12:06:45] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-06-19 12:06:45] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Controller "App\Controller\DefaultController" does neither exist as service nor as class" at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/Controller/ContainerControllerResolver.php line 65 {"exception":"[object] (InvalidArgumentException(code: 0): Controller \"App\\Controller\\DefaultController\" does neither exist as service nor as class at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/Controller/ContainerControllerResolver.php:65, Error(code: 0): Class 'App\\Controller\\DefaultController' not found at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/Controller/ControllerResolver.php:133)"} []
[2018-06-19 12:07:23] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://localhost/symfony_test/public/index.php/","method":"GET"} []
[2018-06-19 12:07:23] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-06-19 12:07:23] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Controller "App\Controller\DefaultController" does neither exist as service nor as class" at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/Controller/ContainerControllerResolver.php line 65 {"exception":"[object] (InvalidArgumentException(code: 0): Controller \"App\\Controller\\DefaultController\" does neither exist as service nor as class at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/Controller/ContainerControllerResolver.php:65, Error(code: 0): Class 'App\\Controller\\DefaultController' not found at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/Controller/ControllerResolver.php:133)"} []
[2018-06-19 12:07:24] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"f05fbf"},"request_uri":"http://localhost/symfony_test/public/index.php/_wdt/f05fbf","method":"GET"} []
[2018-06-19 12:07:42] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://localhost/symfony_test/public/","method":"GET"} []
[2018-06-19 12:07:42] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-06-19 12:07:42] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Controller "App\Controller\DefaultController" does neither exist as service nor as class" at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/Controller/ContainerControllerResolver.php line 65 {"exception":"[object] (InvalidArgumentException(code: 0): Controller \"App\\Controller\\DefaultController\" does neither exist as service nor as class at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/Controller/ContainerControllerResolver.php:65, Error(code: 0): Class 'App\\Controller\\DefaultController' not found at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/Controller/ControllerResolver.php:133)"} []
[2018-06-19 12:08:23] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://localhost/symfony_test/public/","method":"GET"} []
[2018-06-19 12:08:23] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-06-19 12:18:55] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://localhost/symfony_test/public/","method":"GET"} []
[2018-06-19 12:18:55] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-06-19 12:18:58] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://localhost/symfony_test/public/","method":"GET"} []
[2018-06-19 12:18:58] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-06-19 12:21:55] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /" (from "http://localhost/symfony_test/")" at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/EventListener/RouterListener.php line 139 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" (from \"http://localhost/symfony_test/\") at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/EventListener/RouterListener.php:139, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /opt/lampp/htdocs/symfony_test/var/cache/dev/srcDevDebugProjectContainerUrlMatcher.php:161)"} []
[2018-06-19 12:27:46] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://localhost/symfony_test/public/","method":"GET"} []
[2018-06-19 12:27:46] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-06-19 12:28:04] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/EventListener/RouterListener.php line 139 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/EventListener/RouterListener.php:139, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /opt/lampp/htdocs/symfony_test/var/cache/dev/srcDevDebugProjectContainerUrlMatcher.php:161)"} []
[2018-06-19 13:19:25] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/EventListener/RouterListener.php line 139 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /opt/lampp/htdocs/symfony_test/vendor/symfony/http-kernel/EventListener/RouterListener.php:139, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /opt/lampp/htdocs/symfony_test/var/cache/dev/srcDevDebugProjectContainerUrlMatcher.php:161)"} []
[2018-06-19 13:19:25] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"d6a138"},"request_uri":"http://localhost/symfony_test/public/_wdt/d6a138","method":"GET"} []
[2018-06-19 13:19:31] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://localhost/symfony_test/public/lucky/number","method":"GET"} []
[2018-06-19 13:19:31] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-06-19 13:19:31] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"325fba"},"request_uri":"http://localhost/symfony_test/public/_wdt/325fba","method":"GET"} []
[2018-06-19 13:44:41] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://localhost/symfony_test/public/lucky/number","method":"GET"} []
[2018-06-19 13:44:41] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-06-19 13:44:55] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://localhost/symfony_test/public/lucky/number","method":"GET"} []
[2018-06-19 13:44:55] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-06-19 14:08:27] doctrine.DEBUG: SHOW FULL TABLES WHERE Table_type = 'BASE TABLE' [] []
[2018-06-19 14:09:07] doctrine.DEBUG: SHOW FULL TABLES WHERE Table_type = 'BASE TABLE' [] []
[2018-06-19 14:09:07] doctrine.DEBUG: CREATE TABLE migration_versions (version VARCHAR(255) NOT NULL, PRIMARY KEY(version)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB [] []
[2018-06-19 14:09:07] doctrine.DEBUG: SELECT version FROM migration_versions [] []
[2018-06-19 14:09:07] doctrine.DEBUG: SELECT version FROM migration_versions WHERE version IN ('20180619120828') ORDER BY version DESC LIMIT 1 [] []
[2018-06-19 14:09:07] doctrine.DEBUG: SELECT version FROM migration_versions [] []
[2018-06-19 14:09:12] doctrine.DEBUG: "START TRANSACTION" [] []
[2018-06-19 14:09:13] doctrine.DEBUG: CREATE TABLE user (id INT AUTO_INCREMENT NOT NULL, login LONGTEXT NOT NULL, name VARCHAR(255) NOT NULL, surname VARCHAR(255) NOT NULL, email VARCHAR(255) NOT NULL, admin TINYINT(1) NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ENGINE = InnoDB [] []
[2018-06-19 14:09:13] doctrine.DEBUG: INSERT INTO migration_versions (version) VALUES (?) ["20180619120828"] []
[2018-06-19 14:09:13] doctrine.DEBUG: "COMMIT" [] []
[2018-06-19 14:38:33] php.DEBUG: Warning: filemtime(): stat failed for /opt/lampp/htdocs/symfony_test/config/routes/annotations.yaml {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/opt/lampp/htdocs/fedactio_pangea/vendor/symfony/config/Resource/FileResource.php\",\"line\":63,\"trace\":[{\"file\":\"/opt/lampp/htdocs/fedactio_pangea/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php\",\"line\":34,\"function\":\"isFresh\",\"class\":\"Symfony\\\\Component\\\\Config\\\\Resource\\\\FileResource\",\"type\":\"->\"}],\"count\":1})"} []
